### ISSUE TITLE
Support shortcuts in dialogs

### ISSUE THREAD

**Issue by guillep** [pharo-spec/Spec#823]
It would be nice if dialogs respond automatically to enter/escape keys to accept/cancel.
Potentially, it would be nice if those shortcuts are configurable to remove them or replace them for example.

**Comment by estebanlm** [comment]
indeed. In fact this was implemented and I don't know why it was lost.
